src/SFA.DAS.EAS.Portal.Client/PortalClient.cs

```diff
@@ -41,8 +41,15 @@ public PortalClient(IContainer container, IEncodingService encodingService)
 
             // at a later date, we might want to create an empty account doc and add the vacancy details to it, but for now, let's keep it simple
             if (vacanciesTask != null && account != null)
-                account.Vacancies = (await vacanciesTask).ToList();
-            
+            {
+                var vacancies = await vacanciesTask;
+                if (vacancies != null)
+                {
+                    account.VacanciesRetrieved = true;
+                    account.Vacancies = (await vacanciesTask).ToList();
```

Comment:
We could use the vacancies variable which was awaited from the task above

Reply:
Done (the code was wip 😉).